<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminHelpWindow</class>
 <widget class="QDialog" name="AdminHelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Руководство администратора - RayTracing</string>
  </property>
  <widget class="QTreeWidget" name="list_of_contents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Содержание</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Главное меню</string>
    </property>
    <item>
     <property name="text">
      <string>Модуль «Администрирование»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модуль «Теория»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модуль «Практика»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модуль «Тестирование»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дополнительно</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Администрирование пользователей</string>
    </property>
    <item>
     <property name="text">
      <string>Список пользователей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Добавление нового пользователя</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Редактирование пользователя</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пометка на удаление</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Администрирование модуля «Теория»</string>
    </property>
    <item>
     <property name="text">
      <string>Добавление новых файлов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Редактирование выбранных файлов</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Администрирование модуля «Практика»</string>
    </property>
    <item>
     <property name="text">
      <string>Добавление новых работ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Редактирование работ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Проверка выполненных работ</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Администрирование модуля «Тестирование»</string>
    </property>
    <item>
     <property name="text">
      <string>Добавление теста</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Редактирование теста</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Настройки для администратора</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Руководство администратора</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>О программе</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="text_help">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>751</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_prev_page">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/prev-page.png</normaloff>images/prev-page.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_next_page">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/next-page.png</normaloff>images/next-page.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
